src/Form.react.js

```diff
@@ -63,12 +77,7 @@ export default class Form extends Component {
     return this.validate(fieldsToValidate);
   }
 
-  fieldRegister(fieldName, field) {
-    if (field)
-      this.fields[fieldName] = field;
-    else
-      delete this.fields[fieldName];
-  }
+  fields = {}
```

Comment:
move fields after `static defaultProps = {` (~line 33) now it is like in between methods.